ansible/fix-homedir-ownership.yml

```diff
@@ -2,31 +2,17 @@
 - name: Fix Home Directory Ownership
   hosts: all
   gather_facts: true
-  vars_files:
-    - files/admin-oc-networks.yml
   vars:
     # At the time of running this playbook the home directory is not owned by the user.
     # Therefore, we will not be permitted to store the Ansible temporary directory in the home folder.
     # This must be relocated to some where that can be written to by the remote user.
     ansible_remote_tmp: "/tmp/ansible"
   tasks:
-    - name: Ensure hosts are reachable
-      ansible.builtin.command:
-        cmd: "ping -c 1 -w 2 {{ item.value }}"
```

Comment:
Why remove this?

Reply:
Does the ansible terraform provider ensures the hosts are reachable anyway?

Reply:
This is a regression and would make this hard to script. deploy-openstack-config.yml does what you suggest, and we should move that to this playbook since this runs first.